R/get_event_attendees.R

```diff
@@ -1,5 +1,6 @@
 #' Get the attendees for a specified event
 #'
+#' Must be an administrator of the queried Meetup group because of Meetup.com API restriction on get_event_attendees
```

Comment:
Can you link to this: https://www.meetup.com/meetup_api/docs/:urlname/events/:id/attendance/#list
maybe like:

Must be an administrator of the queried Meetup group due to the [Meetup.com API restrictions](https://www.meetup.com/meetup_api/docs/:urlname/events/:id/attendance/#list) on this endpoint.